Pseudo_experiments/generate_random_sampled_experiment.ipynb

```diff
@@ -0,0 +1,1453 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Explore randomly-sampled experiments\n",
+    "\n",
+    "This notebook examines experiments generated by the random sampling approach (sample level simulation).  We use this as a control for our pseudo experiments generated the experiment-preserving approach that is implemented in ```generate_E_GEOD_51409_template_experiment.ipynb```"
```

Comment:
Took me a couple of reads to understand this comment, but this script is you developing the negative control dataset, with generate_E_GEOD_51409_template_experiment is what it's compared to?

Reply:
You're correct! I'm comparing simulated experiments that were simulated differently. I have updated the comment in the notebook so hopefully is makes more sense.